model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java

```diff
@@ -326,7 +334,9 @@ public int getMinimumQuickLoginWaitSeconds() {
 
     @Override
     public void setMinimumQuickLoginWaitSeconds(int val) {
-        setAttribute("minimumQuickLoginWaitSeconds", val);
+        if(val >= 0) {
+            setAttribute("minimumQuickLoginWaitSeconds", val);
```

Comment:
Would not be better to do this in the DefaultExportImportManager (updateRealm and importRealm methods). And I would prefer to throw an exception (ModelException or similar) that will return a 400 bad request for the endpoint. The same for all the properties.

Reply:
Done